deploy-konflux.sh

```diff
@@ -9,8 +9,18 @@ main() {
     echo "Waiting for Konflux to be ready" >&2
     local ret=0
     "${script_path}/wait-for-all.sh" || ret="$?"
-    kubectl describe deployment proxy -n konflux-ui
-    kubectl logs deployment/proxy -n konflux-ui --all-containers=true --tail=10
+    if [ $ret -ne 0 ]; then
+        echo "Deployment failed"
+        ./generate-err-logs.sh
+    else
+        echo -e "\nKonflux is up and running:
+        To access the console, deploy the test resources ./deploy-test-resources.sh
+        Konflux Console is available at https://localhost:9443, you can login using the test user
+        username: user2@konflux.com
+        password: password
+            
+        Follow https://github.com/konflux-ci/konflux-ci?tab=readme-ov-file#enable-pipelines-triggering-via-webhooks to continue the onboarding process\n"
```

Comment:
I'm not sure we should have it here. Few reasons:

1. It duplicates documentation, which will translate to maintenance burden.
2. We provide 3 steps here, each should be done at a different point in time, and that could be confusing to the user.

I think we should just say that Konflux is up and running.

Reply:
I agree, I removed the instructions